packages/types/src/jsonified.ts

```diff
@@ -1,10 +1,10 @@
-import type { JsonValue, Message, PlainMessage } from '@bufbuild/protobuf';
+import type { JsonValue, JsonObject, AnyMessage } from '@bufbuild/protobuf';
 
 // prettier-ignore
 export type Jsonified<T> = T extends JsonValue ? T
```

Comment:
Think this type needs some documentation about what all these extends do

Reply:
doc'd